user_tools/src/spark_rapids_tools/utils/util.py

```diff
@@ -112,6 +112,10 @@ def dump_tool_usage(tool_name: Optional[str], raise_sys_exit: Optional[bool] = T
         sys.exit(1)
 
 
+def get_abfs_account_name(path: str):
+    return path.split('@')[1].split('.')[0]
+
+
```

Comment:
- It is always useful to use type hints to get some checks on type safety and code incorrectness. Python started adopted this typing/atype-annotations more widely to overcome the problem of type-safety that makes python difficult to work with in large projects.
  ```python
   def get_abfs_account_name(path: str) -> str :
  ```
- I don't see `util.py` is the best place for this because it is a functionality that's specific to ADLS driver.
- Should this method try to read the env-variable. if it is not set, then it extracts it from the URL?
- Also, with defining this new method, I would expect 2 sites calling this method: 1- in pytools (old way we xtracted information for Azure); 2- During the initialization in the new spark_rapids_tools
aka, `DBAzureLocalRapidsJob.get_account_name` strips the account-name from the URL; regardless of the initialization we did. It should read

Reply:
Thanks @amahussein! Updated this.